src/components/ModalPayment/ModalAmount/LanternForm.tsx

```diff
@@ -0,0 +1,271 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import { RowFormat, Subheader, LabelText, InputText } from '../styles';
+import { LIGHT_UP_CHINATOWN_TIER_2_LIMIT } from '../consts';
+
+import {
+  useModalPaymentState,
+  useModalPaymentDispatch,
+  ModalPaymentConstants,
+} from '../../../utilities/hooks/ModalPaymentContext';
+
+export const LanternForm = (props) => {
+  const { t } = useTranslation();
+
+  const { amount, lucData } = useModalPaymentState(null);
+  const dispatch = useModalPaymentDispatch(null);
+
+  const [showAddress, setShowAddress] = React.useState(false);
+
+  const subHeader =
+    amount >= LIGHT_UP_CHINATOWN_TIER_2_LIMIT
+      ? 'paymentProcessing.amount.adopt_lantern_tier_3'
+      : 'paymentProcessing.amount.adopt_lantern_tier_2';
+  const subText =
+    amount >= LIGHT_UP_CHINATOWN_TIER_2_LIMIT
+      ? 'paymentProcessing.amount.tier_3_donation'
+      : 'paymentProcessing.amount.tier_2_donation';
+
+  const handleOnChange = (e) => {
+    e.preventDefault();
+    dispatch({
+      type: ModalPaymentConstants.SET_LUC_DATA,
+      payload: {
+        key: e.target.name,
+        value: e.target.value.toString(),
+      },
+    });
+  };
+
+  const handleAddress = (e, bool) => {
+    e.preventDefault();
+    setShowAddress(bool);
+  };
+
+  return (
+    <AmountContainer>
+      <Subheader>{t(subHeader)}</Subheader>
+      <label>{t(subText)}</label>
+      <br />
+      <br />
+      <label>{t('paymentProcessing.amount.personalize')}</label>
+      <br />
+      <br />
+      <SingleRowFormat>
+        <RowFormat width="38%">
+          <LabelText htmlFor="first_name">
```

Comment:
Not too sure, but this `htmlFor` attribute should have a corresponding `id` attribute in the Input element with the same value?

Reply:
i think the name value should be sufficient enough, im not sure what the htmlFor attribute is for anyway, was just following past code.